app/data/repositories/UsersRepository.py

```diff
@@ -30,14 +47,9 @@ def fetchUserByName(username):
 
     c.execute(query)
     
-    user = User(c.fetchone())
-
-    if user.ifUserExists():
-        return user
-
-    return None
-
+    return user(c.fetchone())
 
+# WHAT RETURN WHEN THERE IS NO USER IN DATABASE
```

Comment:
that None looked fine. WHats wrong with it?

Reply:
I simply didn't know how to do this properly.

Reply:
read if it is possible to return optional in python, if not then solve this problem by raising exception.